cms/djangoapps/contentstore/views/library.py

```diff
@@ -115,9 +115,6 @@ def _create_library(request):
     """
     Helper method for creating a new library.
     """
-    if not auth.has_access(request.user, CourseCreatorRole()):
```

Comment:
@mattdrayer I'm not totally clear what that would look like. The outer view method (`library_handler`) does check if libraries are enabled (and the user is logged in) before calling this method, and there are now no other rules.